### PR TITLE
Add Loot Land

### PR DESCRIPTION
**Pull request by VieYang**
## Loot Land

Etherscan link: https://etherscan.io/address/0x8575B7dFE18F4c9A872253bB4f557AD02e94C3A5

Website: https://opensea.io/collection/landproject

Loot Land are parts of the world in Loot metaverse. LAND is divided into parcels that are referenced using unique x,y cartesian coordinates. Feel free to use Loot Land in any way you want.

refer #66

### CODE REVIEW

**Comment by VieYang** [comment]
hi,  @zhew2013 ,  sorry for that of #66 .   Create a new `pull requests` here.

File: utils/lists.ts [hunk 1]
```diff
@@ -467,4 +467,9 @@ export const derivativesList: Record<string, string>[] = [
     description: "The first anti-metaverse built on Ethereum",
     url: "https://upsidedownloot.com/",
   },
+  {
+    name: "Loot Land",
```
Comment (zhew2013) [inline utils/lists.ts:471]: One last thing. Let’s put this entry sorted alphabetically by "name" field just like others. :)

Comment (VieYang) [reply utils/lists.ts:471]: oh, no, ok. I'll update this.

Comment (VieYang) [reply utils/lists.ts:471]: Maybe we need update the `package.json` to add some scripts for those check.

Comment (VieYang) [reply utils/lists.ts]: done.

**Pull request merged**